src/Raven.Server/Json/BlittableJsonTextWriterExtensions.cs

```diff
@@ -1305,12 +1309,20 @@ public static void WriteIndexProgress(this AbstractBlittableJsonTextWriter write
                     writer.WriteInteger(kvp.Value.NumberOfTombstonesToProcess);
                     writer.WriteComma();
 
+                    writer.WritePropertyName(nameof(kvp.Value.NumberOfTimeSeriesDeletedRangesToProcess));
+                    writer.WriteInteger(kvp.Value.NumberOfTimeSeriesDeletedRangesToProcess);
```

Comment:
Doesn't it require studio work? Asking about since `No UI work is needed` was checked.

Reply:
Correct! fixed :)